packages/netlify-plugin-cache-docs/package.json

```diff
@@ -13,7 +13,10 @@
   "license": "MIT",
   "scripts": {},
   "dependencies": {
-    "fs-extra": "^10.1.0"
+    "@googleapis/sheets": "^4.0.2",
```

Comment:
This looks out of context, this is a package to cache deployments
```suggestion
```

Same for the other lines.